traefik/docker-compose-traefik.yml

```diff
@@ -0,0 +1,27 @@
+##############################
+# FILENAME: docker-compose.yml
+# DESCRIPTION: This file is used to configure the docker for Traefix HTTP Reverse Proxy Services
+# USAGE: The file should be placed in /opt/traefik directory. 
+#        It is important that the volume for the SSL Certificates be added and that they correspond with the traefik.toml file in this directory.
+#	   
+##############################
+
+version: '3.4'
+
+services:
+  traefik:
+    image: traefik:latest
```

Comment:
Do we want to pin to a specific image version that we know our config works with?

Reply:
We can do that. I was playing with this and wanted to see if it would pull the latest since there was an update recently.